docs/source/outputs.rst

```diff
@@ -0,0 +1,145 @@
+
+   .. figure:: ../media/S3_out/fig3301.png
+      :alt: Stage 3 background subtracted flux plot
+
+      Stage 3 output: Background Subtracted Flux Plot
+
+   .. figure:: ../media/S3_out/fig3302.png
+      :alt: Stage 3 1-dimensional spectrum plot
+
+      Stage 3 output: 1-Dimensional Spectrum Plot
+
+   .. figure:: ../media/S3_out/fig3303.png
+      :alt: Stage 3 source position fit
+
+      Stage 3 output: Source Position Fit Plot
+
+   .. figure:: ../media/S3_out/fig3305.png
+      :alt: Stage 3 weighting profile
+
+      Stage 3 output: Weighting Profile Plot
+
+   - If ``isplots_S3`` = 5: ``Eureka!`` will plot the Subdata plots from the optimal spectral extraction step.
+
+   .. figure:: ../media/S3_out/fig3501.png
+      :alt: Stage 3 subdata plot
+
+      Stage 3 output: Spectral Extraction Subdata Plot
+   
+.. _s4-out:
+
+Stage 4 Outputs
+---------------
+
+In Stage 4:
+   - If ``isplots_S4`` = 1: ``Eureka!`` will plot the spectral drift per exposure, and the drift-corrected 2-dimensional lightcurve with extracted bins overlaid.
+
+   .. figure:: ../media/S4_out/fig4101.png
+      :alt: Stage 4 spectral drift plot
+
+      Stage 4 output: Spectral Drift Plot
+
+   .. figure:: ../media/S4_out/fig4201.png
+      :alt: Stage 4 2-dimensional binned spectrum
+
+      Stage 4 output: 2-Dimensional Binned Spectrum
+
+   - If ``isplots_S4`` = 3: ``Eureka!`` will plot the spectroscopic lightcurves for each wavelength bin.
+
+   .. figure:: ../media/S4_out/fig4301.png
+      :alt: Stage 4 spectroscopic lightcurve
+
+      Stage 4 output: Spectroscopic Lightcurve
+
+   - If ``isplots_S4`` = 5: ``Eureka!`` will plot the cross-correlated reference spectrum with the current spectrum for each integration, and the cross-correlation strength for each integration.
+
+   .. figure:: ../media/S4_out/fig4401.png
+      :alt: Stage 4 cross correlated reference spectrum
+
+      Stage 4 output: Cross-Correlated Reference Spectrum
+
+   .. figure:: ../media/S4_out/fig4501.png
+      :alt: Stage 4 cross correlation strength
+
+      Stage 4 output: Cross-Correlation Strength
+
+
+.. _s5-out:
+
+Stage 5 Outputs
+---------------
+
+In Stage 5:
+   - If ``isplots_S5`` = 1: ``Eureka!`` will plot the fitted lightcurve model over the data in each channel.
+   
+   .. figure:: ../media/S5_out/fig5101.png
+      :alt: Stage 5 fit data and lightcurve
+
+      Stage 5 output: Fitted lightcurve
+
+   - If ``isplots_S5`` = 3: ``Eureka!`` will plot an RMS deviation plot for each channel to help check for correlated noise, plot the normalized residual distribution, and plot the fitting chains for each channel.
+   .. figure:: ../media/S5_out/fig5501.png
+      :alt: Stage 5 RMS deviation plot
+
+      Stage 5 output: RMS Deviation Plot
+   
+   .. figure:: ../media/S5_out/fig5501.png
+      :alt: Stage 5 residual distribution
+
+      Stage 5 output: Residual Distribution
```

Comment:
One of these must be referring to the wrong file

Reply:
Fixed.